hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md

```diff
@@ -1250,6 +1569,144 @@ can be used:
 Using the explicit endpoint for the region is recommended for speed and the
 ability to use the V4 signing API.
 
+
+## "Timeout waiting for connection from pool" when writing to S3A
```

Comment:
did that everywhere; updated. Also in the troubleshooting s3a memory, just pointed back to the thread tuning entry.